src/main/java/net/library/model/dto/request/UsersTest.java

```diff
@@ -1,11 +1,11 @@
-package net.library.object;
+package net.library.model.dto.request;
 
-public class User {
+public class UsersTest {
```

Comment:
What is this, and why is this class with the test prefix lying in the general folder?

Reply:
+